### PR TITLE
#8245: Map with mapbox or mapboxstyle background layers break the app when saved

### PR DESCRIPTION
**Pull request by allyoucanmap**
## Description

include access token to mapbox background layers

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

## Issue

**What is the current behavior?**

#8245

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [x] No

### CODE REVIEW

File: web/client/utils/ConfigUtils.js [hunk 1]
```diff
@@ -169,6 +169,10 @@ export const setApiKeys = function(layer) {
     if (layer.type === 'mapquest') {
         layer.apiKey = defaultConfig.mapquestApiKey;
     }
+    if (layer.type === 'tileprovider' && ['MapBoxStyle', 'MapBox'].includes(layer.provider)) {
+        // include an empty string if missing to avoid errors in the layer url template
+        layer.accessToken = defaultConfig.mapboxAccessToken || '';
```
Comment (MV88) [inline web/client/utils/ConfigUtils.js:174]: what if you include this in the defaultConfig object as a default value?

similar to what is done with mapquestApiKey, so you can skip || '' I suppose

**Review by MV88 — CHANGES_REQUESTED** [review]
@allyoucanmap let me know what  you think about this

File: web/client/utils/ConfigUtils.js (hunk 1 above)
Comment (allyoucanmap) [reply web/client/utils/ConfigUtils.js:174]: thanks @MV88 , I'll check it and replace

**Review by MV88 — APPROVED** [review]

**Comment by MV88** [comment]
@ElenaGallo please test in DEV

**Comment by ElenaGallo** [comment]
Test passed, @allyoucanmap please backport

**Pull request merged**